src/server-rm/pom.xml

```diff
@@ -106,6 +106,23 @@
             </plugins>
             </build>
         </profile>
+	    <profile>
+			<!-- some servlet containers (Tomcat) does not ship with a JSTL impl
+			In such cases, run with this profile-->
+	        <id>with-jstl-impl</id>
+	        <dependencies>
+				<dependency>
+		            <groupId>jakarta.servlet.jsp.jstl</groupId>
+		            <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
```

Comment:
Is this needed? I think the impl below will ensure api classes are present.

Reply:
It is needed. Don't ask my why, but when I remove "jakarta.servlet.jsp.jstl-api" (but keep the impl, I get the following at runtime when trying to get a jsp page.

[INFO] Apr 01, 2024 3:14:40 PM org.apache.catalina.core.ApplicationDispatcher invoke
[INFO] SEVERE: Servlet.service() for servlet [jsp] threw exception
[INFO] java.lang.ClassNotFoundException: jakarta.servlet.jsp.jstl.core.ConditionalTagSupport
[INFO] 	at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1353)
[INFO] 	at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1165)
[INFO] 	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
[INFO] 	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
[INFO] 	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)